src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -287,93 +283,6 @@ export class JupyterNotebookBase implements INotebook {
         return this.session ? this.session.waitForIdle(timeoutMs) : Promise.resolve();
     }
 
```

Comment:
Intialize code (working dir / startup commands / plot init) now all moved from Notebook into Kernel.

Reply:
Gave a quick peek with notebooks and IW. Startup command, new working directory, and plots all looked good after a notebook start as well as after a kernel restart.

Reply:
@DonJayamanne could you take a re-look? It's still blocked on the change requested.